extensions/amp-analytics/amp-analytics.md

```diff
@@ -87,6 +89,37 @@ At the top level, the config has following properties:
     -  **vars**: This property is same as the property of same name in parent block. Values specified here take precedence over the values specified in the platform and the parent block. 
 
 
+##### <a name="vars-substitution"></a>Variable Substitution and Formatting
+Variables can be specified in multiple places in the analytics config. Some of the variables are predefined by the platform. In addition, variables can also be defined, overridden or cleared by the config. Values set at the top level get applied to all the triggers and the values specified inside a trigger only apply to that trigger. The values of variables get encoded before the substitution using `encodeURIComponent`. The precedence for variable values is in the order trigger.vars > config.vars > platform.vars.
```

Comment:
I did this but while doing this, I realized that this is leading to a lot of duplication that is not really needed. I ended up writing just a one line for each variable. If you think we should expand on each of the variables as is done in the other doc, let me know.